en/docs/try-out/account-and-transaction-flow.md

```diff
@@ -300,24 +308,24 @@ user that has a `subscriber` role.
 
 #### Consent Re-authentication for Account Consents
 
- Account consents are long-lived consents and PSUs can re-authenticate account consents. Consent re-authentication is 
- the process that enables an ASPSP to authenticate a PSU more than once for the same consent. To re-authenticate a consent:
+ Account consents are long-lived consents and bank customer can re-authenticate account consents. Consent re-authentication is 
+ the process that enables an Data Holder to authenticate a customer more than once for the same consent. To re-authenticate a consent:
  
  - The consent must be in the authorized state.
  - The `ExpirationDateTime` of the consent should not have elapsed.
  
- Once re-authentication is successful, an AISP must not use or refresh access tokens that were issued for the same consent. 
- The ASPSP can decide to invalidate the previously issued tokens for the same consent. 
+ Once re-authentication is successful, an Accredited Data Recipient must not use or refresh access tokens that were issued for the same consent. 
+ The Data Holder can decide to invalidate the previously issued tokens for the same consent. 
  
 ??? tip "Changing authorized accounts during re-authentication"
 
-     In the Account and transaction flow, the PSU can change the account IDs related to a consent during consent re-authentication. 
-     The ASPSP can allow the PSU to change the account IDs using the following configurations:
+     In the Account and transaction flow, the bank customer can change the account IDs related to a consent during consent re-authentication. 
+     The Data Holder can allow the bank customer to change the account IDs using the following configurations:
      
      1. Open the `<IS_HOME>/repository/conf/deployment.toml` file.
      2. Set the following configuration to `true`:
         ``` toml
-        [open_banking_uk.consent]
+        [open_banking_cds.consent]
         acc_update_by_psu_enabled = false
```

Comment:
as we ask the users to set this to `true`, let's change to `acc_update_by_psu_enabled = true`